src/cpp/include/openvino/genai/vlm_pipeline.hpp

```diff
@@ -65,37 +65,14 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
     explicit VLMPipeline(
         const std::filesystem::path& model_dir,
         const std::string& device="CPU",
-        const ov::AnyMap device_config={},
-        ov::Core core=ov::Core{}
-    ) : VLMPipeline{
-        model_dir,
-        Tokenizer(model_dir.string(), device_config),
-        device,
-        device_config,
-        core
-    } {}
-
-    /// @brief Construct a pipeline form a folder containing model IRs
-    /// and from a Tokenizer instance.
-    /// @param model_dir A folder to read model IRs.
-    /// @param tokenizer An instance of Tokenizer to use.
-    /// @param device Inference device.
-    /// @param device_config A config to pass to ov::Core.set_property()
-    /// and ov::Core::compile_model().
-    /// @param core ov::Core instance to use.
-    VLMPipeline(
-        const std::filesystem::path& model_dir,
-        const ov::genai::Tokenizer& tokenizer,
-        const std::string& device="CPU",
-        const ov::AnyMap device_config={},
-        ov::Core core=ov::Core{}
+        const ov::AnyMap device_config={}
     );
 
     /// @brief Default destructor.
     ~VLMPipeline();
 
     /// @brief Generate a response given a prompt and any number of
-    /// uint8 RGB images.
+    /// uint8 RGB images with [HWC] layout.
```

Comment:
if I pass NHWC, what will happen?

Reply:
1HWC is actually expected. Should I enable both?

Reply:
I suppose yes, maybe in PR with multiple images you can add support of NHWC ? where ov::Tensor represents multiple images and N > 1

Reply:
Done in [7441a18](https://github.com/Wovchena/openvino.genai-public/pull/68/commits/7441a1860def9dfb8012ef976b0d8526ed4975d6). @popovaan, please, add tests for [NHWC] ([1HWC], [2HWC]) and [HWC] image layouts.